app/controllers/login_controller.rb

```diff
@@ -1,8 +1,14 @@
 class LoginController < ApplicationController
   Mumukit::Login.configure_login_controller! self
 
+  skip_before_action :verify_authenticity_token
```

Comment:
Hmm, but with this change, we are never going to verify the token, even when we can :/